src/cloud-api-adaptor/podvm/README.md

```diff
@@ -77,31 +77,22 @@ $ docker build -t podvm_builder \
 
 ## Building the image containing the podvm binaries
 
-In order to build the binaries you should be using the corresponding dockerfile
-of the Linux distro for which the builder image was built.
-For example, if the builder image was built with *Dockerfile.podvm_builder.DISTRO* then
-you should use the *Dockerfile.podvm_binaries.DISTRO* to build the image with podvm binaries.
+We have make targets handy for you. Make sure you are under the
+`src/cloud-api-adaptor` folder.
 
-Assuming you have built the podvm_builder image for Ubuntu as explained in the previous step,
-running the following command will build the image with the podvm binaries.
+> **Note:** The `BUILDER_IMG` environment variable is crucial as it specifies
+> the builder image, which is the result of the previous step build. Ensure you
+> have built the builder image before proceeding.
+
+To build the binaries image, use the following command:
 
 ```bash
-$ docker build -t podvm_binaries \
-         --build-arg BUILDER_IMG=podvm_builder \
-         -f Dockerfile.podvm_binaries .
+$ BUILDER_IMG=<your_builder_image> make podvm-binaries
 
-```
```

Comment:
This line needs reinstating to close the code block

Reply:
nice catch, thanks. fixed.